telco-core/configuration/reference-crs-kube-compare/required/networking/metallb/addr-pool.yaml

```diff
@@ -0,0 +1,20 @@
+# required
```

Comment:
The metallb resources must be optional. Not all customers / partners will be using BGP peering with metallb for their core deployments